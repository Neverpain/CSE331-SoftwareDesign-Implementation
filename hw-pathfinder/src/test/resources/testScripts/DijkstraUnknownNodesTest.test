# Tests Dijkstra's algorithm on unknown nodes, prints out
# output

CreateGraph g

AddNode g A
AddNode g B

FindPath g A X

FindPath g B X

FindPath g X X

FindPath g X Y

FindPath g Y X