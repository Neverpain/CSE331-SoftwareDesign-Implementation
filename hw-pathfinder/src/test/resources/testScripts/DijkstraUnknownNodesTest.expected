# Tests Dijkstra's algorithm on unknown nodes, prints out
# output

created graph g

added node A to g
added node B to g

unknown: X

unknown: X

unknown: X
unknown: X

unknown: X
unknown: Y

unknown: Y
unknown: X